src/coreclr/jit/jitmetadatalist.h

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// clang-format off
+
+#ifdef JITMETADATA
+#define JITMETADATAINFO(name, type, flags) JITMETADATA(name, type, flags)
+#define JITMETADATAMETRIC(name, type, flags) JITMETADATA(name, type, flags)
+#endif
+
+#if !defined(JITMETADATAINFO) || !defined(JITMETADATAMETRIC)
+#error Define JITMETADATAINFO and JITMETADATAMETRIC before including this file.
+#endif
+
+//              Name,                          type              flags
+JITMETADATAINFO(MethodFullName,                const char*,      0)
+JITMETADATAINFO(TieringName,                   const char*,      0)
+JITMETADATAMETRIC(PhysicallyPromotedFields,    int,              0)
+JITMETADATAMETRIC(LoopsFoundDuringOpts,        int,              0)
+JITMETADATAMETRIC(LoopsCloned,                 int,              0)
+JITMETADATAMETRIC(LoopsUnrolled,               int,              0)
+JITMETADATAMETRIC(LoopAlignmentCandidates,     int,              0)
+JITMETADATAMETRIC(LoopsAligned,                int,              0)
+JITMETADATAMETRIC(VarsInSsa,                   int,              0)
+JITMETADATAMETRIC(HoistedExpressions,          int,              0)
+JITMETADATAMETRIC(RedundantBranchesEliminated, int,              JIT_METADATA_HIGHER_IS_BETTER)
```

Comment:
Are the flags used anywhere? Where is `JIT_METADATA_HIGHER_IS_BETTER` defined? It seems odd to include this file in superpmi source code but not have this defined there.

Reply:
They aren't yet, but my plan was to use this to automatically colorize the metrics in the report (once I add that support). Indeed I haven't defined the enum anywhere yet (and it'll probably only end up being define in superpmi and not the JIT).